### PR TITLE
B3 View / Manage Lookup Tables

### PR DESCRIPTION
**Pull request by biyeun**
screen shot 2016-02-18 at 2 42 02 pm
screen shot 2016-02-18 at 2 42 14 pm
screen shot 2016-02-18 at 2 42 34 pm
screen shot 2016-02-18 at 2 42 48 pm

### CODE REVIEW

**Comment by biyeun** [comment]
QA ticket submitted and feature is on staging

**Comment by millerdev** [comment]
Reviewed quickly

**Comment by biyeun** [comment]
QA complete (!!!!) good to merge cc @benrudolph

File: corehq/apps/fixtures/static/fixtures/js/lookup-manage.js [hunk 1]
```diff
@@ -218,6 +220,15 @@ $(function () {
         self.file = ko.observable();
         self.selectedTables = ko.observableArray([]);
 
+        self.removeBadDataType = function (dataType) {
+            setTimeout(function () {
+                // This needs to be here otherwise if you remove the dataType
+                // directly from the dataType, the DOM freezes and the page
```
Comment (benrudolph) [inline corehq/apps/fixtures/static/fixtures/js/lookup-manage.js]: remove the dataType directly from the dataType?

**Comment by benrudolph** [comment]
nice this is big!! :heart_eyes_cat: :heart_eyes_cat:

**Comment by czue** [comment]
:tada:

File: corehq/apps/fixtures/static/fixtures/js/lookup-manage.js (hunk 1 above)
Comment (biyeun) [inline corehq/apps/fixtures/static/fixtures/js/lookup-manage.js]: i could have worded that a bit better huh

**Pull request merged**